### DISCUSSION TITLE
Numpad does not work properly

### DISCUSSION THREAD

**Discussion by Andi2304** [apohl79/audiogridder#917] [Help]
I‘m running Pro Tools on a Mac. Typing numbers on the numpad (Plugin on DAW side) does not work. I get a letter instead of a number. The numbers on the left of the keyboard do work though.

On the Server side it works.

Thanks

Andi

**Comment by Andi2304** [comment]
On Beta 14 the Numpad worked. Now on Beta 15 it is broken again. At least on my system.

Best

Andi

**Comment by apohl79** [comment]
Ok, thx for reporting this!
